lib/interface_quda.cpp

```diff
@@ -4891,6 +5372,14 @@ void copyExtendedResidentGaugeQuda(void *resident_gauge)
   static_cast<GaugeField *>(resident_gauge)->copy(*extendedGaugeResident);
 }
 
+//void performGaussianSmearNStep(void *h_out, void *h_in, QudaInvertParam *param, const int n_steps, const double omega){
```

Comment:
I see the legacy functions are still defined below.  Do these thin wrappers not work as expected yet, or can we delete the below and switch to these?

Reply:
removed in 6dfde4caaaa043937d1e3f38fffff6ea40752245

Reply:
additional object removed in f91077839edb17b7c925be3b72eae8769e687a14